### PR TITLE
Docker demo more direct

### PR DESCRIPTION
**Pull request by batmat**
Built above #214 not yet merged

Essentially:
* Disable the upgrade wizard
* Add Welcome message + link to the /blue URL
* Install some additional plugins to possibly make the experience more straightforward

### CODE REVIEW

**Comment by batmat** [comment]
To try it easily, if need be:

    docker run -p 8080:8080 batmat/blueocean-demo

**Comment by batmat** [comment]
Ping @i386 @michaelneale and @ndeloof as I'm building on your work.

**Comment by ndeloof** [comment]
I've added your commit on top of https://github.com/jenkinsci/blueocean-plugin/pull/214

**Comment by pskumar448** [comment]
@batmat 
The docker image of the blueocean-demo Jenkins has security enable, Could you share the details.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+FROM jenkinsci/jenkins:2.6
```
Comment (rtyler) [inline Dockerfile]: Any reason not to just use the `latest` tag?

Comment (ndeloof) [inline Dockerfile]: no specific reason

**Comment by batmat** [comment]
Closing as @ndeloof added the commit in #214.

**Pull request closed without merging**